package.json

```diff
@@ -32,7 +32,8 @@
   "pnpm": {
     "patchedDependencies": {
       "semantic-release-monorepo@7.0.5": "patches/semantic-release-monorepo@7.0.5.patch",
-      "@open-wc/lit-helpers@0.5.1": "patches/@open-wc__lit-helpers@0.5.1.patch"
+      "@open-wc/lit-helpers@0.5.1": "patches/@open-wc__lit-helpers@0.5.1.patch",
+      "wc-storybook-helpers@1.3.1": "patches/wc-storybook-helpers@1.3.1.patch"
```

Comment:
This is the only thing that I am not a 100% sure with. I know pnpm patches sound like a great idea, however, you could also add the changed code to `packages/docs/src/helpers/component.ts` into the storybookDefaults function. This would have made it possible to easily upgrade `@open-wc/lit-helpers` when there is a new release, which unfurtunately is not possible currently.

Reply:
I understand the perspective. I tried to have the following approach:
1. Things that are buggy are patched. (https://github.com/synergy-design-system/synergy-design-system/pull/343/files#diff-766d360edf31dd65886b2efed53ffdc8cad96fb8577d6c2913d1f5af7847b779R69-R84)
2. Things that would have changes in the helper AND would have to be patched are only patched (e. g. https://github.com/synergy-design-system/synergy-design-system/pull/343/files#diff-766d360edf31dd65886b2efed53ffdc8cad96fb8577d6c2913d1f5af7847b779R101-R103)
3. Things that are easily addable/overridable on our side are added in the helper, e. g. https://github.com/synergy-design-system/synergy-design-system/pull/343/files#diff-b0ab8fca684831c354518633163bf08d0188724b7b25440a39f92fd37b104421R100-R104

For the first two issues would try to make a PR for the original package. As the author already took some other things from my fork, I believe this will be quite lightweight to bring in. I would make a follow up issue for that.

Fine for you?